lib/src/screens/settings/connection_sync_page.dart

```diff
@@ -28,6 +30,14 @@ class ConnectionSyncPage extends BasePage {
 
   @override
   Widget body(BuildContext context) {
+    var showRescan = dashboardViewModel.hasRescan;
+    if (dashboardViewModel.wallet.type == WalletType.decred &&
+        DecredWalletBase.walletBirthdayBlockHeight(
```

Comment:
I'm not too sure about referencing `DecredWalletBase` directly from this file. This code will fail to build if decred is excluded.

Reply:
oh yeah, I think that's right, can't import from cw_decred directly. need to add it to the wallet interface I think. I can do this if you'd like, or show you how.

Reply:
> I'm not too sure about referencing `DecredWalletBase` directly from this file. This code will fail to build if decred is excluded.

Ah, I see.